exporter/elasticsearchexporter/README.md

```diff
@@ -0,0 +1,82 @@
+# Elasticsearch Exporter
+
+This exporter supports sending OpenTelemetry logs to [Elasticsearch](https://www.elastic.co/elasticsearch).
+
+## Configuration options
+
+- `urls`: List of Elasticsearch URLS. If urls and cloudid is missing the
```

Comment:
```suggestion
- `urls`: List of Elasticsearch URLs. If urls and cloudid is missing, the
```

Reply:
Thank you. Will rename the setting to `endpoints` though.